Bio/Align/AlignInfo.py

```diff
@@ -594,6 +595,6 @@ def print_info_content(summary_info, fout=None, rep_record=0):
     fout = fout or sys.stdout
     if not summary_info.ic_vector:
         summary_info.information_content()
-    rep_sequence = summary_info.alignment[rep_record].seq
-    for pos, ic in enumerate(summary_info.ic_vector):
-        fout.write("%d %s %.3f\n" % (pos, rep_sequence[pos], ic))
+    rep_sequence = summary_info.alignment[rep_record]
+    for pos, (aa, ic) in enumerate(zip(rep_sequence, summary_info.ic_vector)):
+        fout.write("%d %s %.3f\n" % (pos, aa, ic))
```

Comment:
I assume that since this relies on iterating over a SeqRecord giving single letters, then the ``.seq`` is not needed. In fact, is there any reason to keep the local variable ``rep_sequence``?

Reply:
We don't need `rep_sequence` as a local variable. But the code is a bit more readable with this local variable instead of stuffing everything on line 599.

But I am OK either way. If you prefer to remove the local variable, I can do so.

Reply:
Not too bothered, as is is fine.